go/interpreter/lfvm/instructions_test.go

```diff
@@ -1497,3 +1497,35 @@ func TestGenericCall_ProperlyReportsErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestGenericCall_CallKindPropagatesStaticMode(t *testing.T) {
```

Comment:
I am afraid this test does not test the property.

Calling CALL in static mode is promoted to StaticCall 
I guess this can be checked through the parameters of the mock. 
Test only needs to test CALL